### ISSUE TITLE
Failed to spawn: bootstrapper crashed with signal 11 at offset 0x1514; please file a bug

### ISSUE THREAD

**Issue by klmitchell2** [frida/frida#2738]
Hello, 

I'm running the latest version of Frida on my machine (Apple M1 Max) and on my device (Pixel 5 API 30) and I'm receiving this error intermittently.

```
frida-sandbox frida -U -f com.google.android.calendar
     ____
    / _  |   Frida 16.1.10 - A world-class dynamic instrumentation toolkit
   | (_| |
    > _  |   Commands:
   /_/ |_|       help      -> Displays the help system
   . . . .       object?   -> Display information about 'object'
   . . . .       exit/quit -> Exit
   . . . .
   . . . .   More info at https://frida.re/docs/home/
   . . . .
   . . . .   Connected to Android Emulator 5554 (id=emulator-5554)
Failed to spawn: bootstrapper crashed with signal 11 at offset 0x1514; please file a bug
         pc: 00000072bf79b514
         lr: 00000072bf79aab4
         sp: 0000007fcf451be0
         x0: 0000007019375754
         x1: 00000072bf79bb91
         x2: 0000000000000001
         x3: 0000000000000002
         x4: 0000000000000028
         x5: 0000000000000000
         x6: b40000703bf78ed0
         x7: 0000000000000000
         x8: 0000000000000002
         x9: 0000000000000002
        x10: 0000000000000001
        x11: 0000007019373f70
        x12: 0000000000000001
        x13: 0000000000000001
        x14: 0000000000000000
        x15: 0000000029aaaaab
        x16: 00000070238fabc8
        x17: 00000072c0b1cd64
        x18: 00000072c1e50000
        x19: 0000007fcf451c98
        x20: 0000007fcf451c18
        x21: 0000007019375754
        x22: 0000000000000028
        x23: 000000000015ffb0
        x24: 0000007019214000
        x25: 000000701937386d
        x26: 00000000000023c0
        x27: 00000070193005b8
        x28: 431bde82d7b634db
        x29: 0000007fcf451be0
``` 

I've also experienced this error running on an Intel MBP and a Physical Pixel 4 device running the latest versions of Frida.

**Comment by Monsieur-Chat** [comment]
Same for me

**Comment by klmitchell2** [comment]
@Monsieur-Chat - Can you confirm if you are using 16.1.10?

**Comment by Monsieur-Chat** [comment]
Yes latest version tested on Androind using Genymotion and tested on Android API 31 ,30 and 29

**Comment by klmitchell2** [comment]
I reverted to 16.1.0 and no longer experienced this issue.

**Comment by Monsieur-Chat** [comment]
Thx for information i will try this

**Comment by Jesbinder** [comment]
I am running version 16.1.10, the issue still persists.

**Comment by christiancoleman** [comment]
I'm getting the same error. I tried reverting to 16.1.9 and it still happened. In 16.1.8 it works like it used to i.e. without errors.

**Comment by christiancoleman** [comment]
In my case I was working in an arch64 environment. Looking at the commit for 16.1.9 it seems this file might be the culprit:

https://github.com/frida/frida-core/compare/d8ea44810bb5e61010b7cd2b0bda135780cc807b...722b1ffc31601f8641163c04b936368a66d2da0f#diff-2108fbb597624f9b54e323a05dbc19dc6498a8869e61e6da3e54dbd3a23849ed

A bit more info is I'm using a rooted ASUS ROG II which is on Android 11.
Uname -a is: Linux localhost 4.14.190-perf #1 SMP PREEMPT Tue Mar 8 00:02:53 CST 2022 aarch64

**Comment by trimmytech** [comment]
i am having the same issue

**Comment by Rhynorater** [comment]
I'm also having this issue

**Comment by MrWildanMD** [comment]
same issue using apple m1 and testing to real device android 10

**Comment by mangustik228** [comment]
Same problem
frida-tools 16.1.10. 
Ubuntu 22.04. 
Genymotion / Android 10

**Comment by dem0ns** [comment]
same, and phone crashed with a fake reboot, adb still connected and frida-server still running 

Android 10

**Comment by PangorroHammer** [comment]
Same Error
Frida-server 16.1.10
Moto G9
Android 11 -arm 64
Kali Linux

**Comment by datph4m253** [comment]
Same issue

frida 16.1.10

redmi note 8 

windows 10

**Comment by oleavr** [comment]
Thanks for reporting! Fixed in latest git. Fixes will be included in 16.1.11, to be released soon.

**Comment by MrWildanMD** [comment]
ok guys the bootstrapper crash error is gone but now i'm encountering timed out when try to spawn a process
